spec/features/state_file/prior_year_access_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require 'axe-capybara'
+require 'axe-rspec'
+
+RSpec.feature "accessing a prior year PDF" , js: true, active_job: true do
+  before do
+    allow_any_instance_of(Routes::StateFileDomain).to receive(:matches?).and_return(true)
+    Flipper.enable(:get_your_pdf)
+  end
+
+  it "has content" do
```

Comment:
this could say a little more, for example `it "allows a user to enter an email address and offboards them when they enter an incorrect code"` (if that's what's happening here?)